feature/experimental/gribi/otg_tests/vrf_policy_driven_te/README.md

```diff
@@ -0,0 +1,839 @@
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_222"
+                            }
+                        }
+                        rule {
+                            sequence-id: 11
+                            ipv4 {
+                                protocol: 4
+                                source-address: "ipv4_outer_src_111"
+                            }
+                            action {
+                                decap-network-instance: "DECAP_TE_VRF"
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_111"
+                            }
+                        }
+                        rule {
+                            sequence-id: 12
+                            ipv4 {
+                                protocol: 41
+                                source-address: "ipv4_outer_src_111"
+                            }
+                            action {
+                                decap-network-instance: "DECAP_TE_VRF"
+                                post-network-instance: "DEFAULT"
+                                decap-fallback-network-instance: "TE_VRF_111"
+                            }
+                        }
+                        rule {
+                            sequence-id: 13
+                            ipv4 {
+                                dscp-set: [dscp_encap_a_1, dscp_encap_a_2]
+                            }
+                            action {
+                                network-instance: "ENCAP_TE_VRF_A"
+                            }
+                        }
+                        rule {
+                            sequence-id: 14
+                            ipv6 {
+                                dscp-set: [dscp_encap_a_1, dscp_encap_a_2]
+                            }
+                            action {
+                                network-instance: "ENCAP_TE_VRF_A"
+                            }
+                        }
+                        rule {
+                            sequence-id: 15
+                            ipv4 {
+                                dscp-set: [dscp_encap_b_1, dscp_encap_b_2]
+                            }
+                            action {
+                                network-instance: "ENCAP_TE_VRF_B"
+                            }
+                        }
+                        rule {
+                            sequence-id: 16
+                            ipv6 {
+                                dscp-set: [dscp_encap_b_1, dscp_encap_b_2]
+                            }
+                            action {
+                                network-instance: "ENCAP_TE_VRF_B"
+                            }
+                        }
+                        rule {
+                            sequence-id: 17
```

Comment:
Can all the comments from https://github.com/openconfig/featureprofiles/pull/2217 be addressed in the readme file.